src/LDNS.xs

```diff
@@ -2459,6 +2459,18 @@ rr_cname_cname(obj)
     CLEANUP:
         free(RETVAL);
 
+MODULE = Zonemaster::LDNS        PACKAGE = Zonemaster::LDNS::RR::DNAME            PREFIX=rr_dname_
+
+char *
+rr_dname_dname(obj)
```

Comment:
suggestion: name the method `target` instead, as it seems to be the given name of the RDATA [RFC 6672 section 2](https://www.rfc-editor.org/rfc/rfc6672#section-2).

Reply:
Oh I understand. This is totally fine then.